api/src/test/java/com/epam/pipeline/acl/metadata/MetadataApiServiceTest.java

```diff
@@ -85,19 +111,37 @@ public void shouldUpdateMetadataItemKeyForAdmin() {
 
     @Test
     @WithMockUser
-    public void shouldUpdateMetadataItemKeyWhenPermissionIsGranted() {
+    public void shouldUpdateMetadataItemKeyForOwner() {
         doReturn(metadataEntry).when(mockMetadataManager).updateMetadataItemKey(metadataVO);
-        doReturn(entity).when(mockEntityManager).load(ENTITY_ACL_CLASS, ID);
+        mockLoadEntity(entityWithOwner, ID);
         mockAuthUser(SIMPLE_USER);
 
         assertThat(metadataApiService.updateMetadataItemKey(metadataVO)).isEqualTo(metadataEntry);
     }
 
+    @Test
+    @WithMockUser(username = SIMPLE_USER)
+    public void shouldUpdateMetadataItemKeyForPipelineUser() {
```

Comment:
Let's add the similar test to check if an admin is able to perform the operation for pipeline user metadata.